CHANGES.md

```diff
@@ -68,6 +68,7 @@
 
 ## Breaking Changes
 
+* Python SDK now does not allow mixing the yield and return statements in `DoFn.process()` ([#22969](https://github.com/apache/beam/issues/22969)). `yield` is recommended for emitting elements and `yield from` for iterators.
```

Comment:
```suggestion
* Python SDK now does not allow mixing the yield and return statements in `DoFn.process()` ([#22969](https://github.com/apache/beam/issues/22969)). We recommend to use `yield`  for emitting individual elements and `yield from` for emitting the content of entire iterables.
```

Reply:
this suggestion should be added to Beam docs.

Reply:
I added this to the warning message. The current doc indeed has this information here: https://beam.apache.org/documentation/programming-guide/#pardo